Makefile

```diff
@@ -305,6 +305,14 @@ bundle-update: verify-previous-version ## Update CSV fields and validate the bun
 	sed -r -i "s|replaces: .*|replaces: machine-deletion-remediation.v${REPLACES_VERSION}|;" ${CSV}
 	$(MAKE) bundle-validate
 
+.PHONY: bundle-reset
+bundle-reset: ## Revert all version or build date related changes
```

Comment:
Is there any other target that will use it or is it just for manual triggering?

Reply:
Would it make sense to just call this target at the end of each `bundle` target call or simply call `make bundle`? I think the second option was just to revert any CSV changes from  `bundle-update`, and IIRC we run the `bundle` target before each bundle creation (using the related target)

Reply:
what about calling it in `verify-no-changes`? It would catch e.g. accidentally added `creationDate` because of modified file 😉 Proofed to be useful in NHC for a while already

Reply:
good idea :+1: